pkg/plugin/aqua/client/client.go

```diff
@@ -68,6 +95,7 @@ func NewClient(baseURL string, authorization Authorization) *Client {
 		authorization: authorization,
 		httpClient:    httpClient,
 	}
+	client.setAuthHeader()
```

Comment:
maybe its better to take out auth setting from client constructor , so error can be handled

Reply:
It would be easier to read this code if we renamed setAuthHeader to getAuthHeader and let it return an error for proper error handling. Then we can implement the fallback logic, e.g. default to Authorization header. It's also fine to let NewClient return the error.

BTW, is it going to work if we fallback to Authorization header?